scripts/lib/sphinx/sphinxHtmlToMarkdown.ts

```diff
@@ -193,7 +193,8 @@ export async function sphinxHtmlToMarkdown(options: {
       .map((child) => {
         const $child = $page(child);
         $child.find(".viewcode-link").closest("a").remove();
-        const id = $dl.find("dt.sig-object").attr("id");
+        const findID = $dl.find("dt.sig-object").attr("id");
+        const id = findID != undefined ? findID : "";
```

Comment:
I believe this will work:

```suggestion
        const id = $dl.find("dt.sig-object").attr("id") || "";
```

Reply:
That's better!